code/modules/spells/spell_types/pointed/duffelbag_curse.dm

```diff
@@ -45,10 +45,13 @@
 			if(!target.put_in_hands(C))
 				target.dropItemToGround(target.get_active_held_item())
 				target.put_in_hands(C)
+
 /obj/effect/proc_holder/spell/pointed/duffelbagcurse/can_target(atom/target, mob/user, silent)
 	. = ..()
 	if(!.)
 		return FALSE
+	if(!isInSight(target, user))
+		return FALSE
```

Comment:
Check if the target is in `get_hear()`, it's like view() but ignores darkness.

Reply:
isnt that dview

Reply:
```dm
//Version of view() which ignores darkness, because BYOND doesn't have it (I actually suggested it but it was tagged redundant, BUT HEARERS IS A T- /rant).
/proc/dview(range = world.view, center, invis_flags = 0)

```

Reply:
I keep forgetting

Reply:
dview doesnt prevent xraying